example/pkg/apis/miskatonic/v1beta1/university_types.go

```diff
@@ -26,7 +26,7 @@ import (
 // +genclient=true
 
 // +k8s:openapi-gen=true
-// +resource=universities
+// +resource=universities,University
```

Comment:
This needs to be UniveristyREST if you want it to call `NewUniversityREST()` in the code generator.